lib/lib.rs

```diff
@@ -176,6 +176,18 @@ pub fn read_deploy_file<P: AsRef<Path>>(deploy_path: P) -> Result<Deploy, Error>
     read_deploy(Cursor::new(input))
 }
 
+/// Reads a previously-saved [`Transaction`] from a file.
+pub fn read_transaction_file<P: AsRef<Path>>(transaction_path: P) -> Result<TransactionV1, Error> {
```

Comment:
does this need to be public?

Reply:
I gave this function the same level of exposure as the other functions here, similar to the read_deploy_file function. I would argue it needs to be public as the lib should support this functionality. Additionally, the client itself needs to pull this function from the lib.

Reply:
:+1: